README.pod

```diff
@@ -140,6 +144,8 @@ The format argument allows the following interpreted sequences:
 
   %N    required by
 
+  %W    optional for
+
```

Comment:
Same here

Reply:
Of course, see 9549a061a16dfd9bebb3fedb518abe96245f5828.